toolkit/sidechain/domain/src/lib.rs

```diff
@@ -797,27 +870,41 @@ impl Display for McBlockHash {
 	}
 }
 
+/// Extended information about an (unspent) transaction output in Cardano ledger
 #[derive(
 	Default, Debug, Copy, Clone, PartialEq, Eq, Encode, Decode, DecodeWithMemTracking, TypeInfo,
 )]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
 pub struct UtxoInfo {
+	/// Output ID
 	pub utxo_id: UtxoId,
+	/// Epoch number in which the output was produced
 	pub epoch_number: McEpochNumber,
+	/// Block number in which the output was produced
 	pub block_number: McBlockNumber,
+	/// Slot number in which the output was produced
 	pub slot_number: McSlotNumber,
+	/// Index in block of the transaction that produced the output
 	pub tx_index_within_block: McTxIndexInBlock,
 }
 
+/// Key type used for ordering transaction outputs
+///
+/// This ordering key is used in contexts where a common ordering of the data must be used
+/// by all nodes participating in a Partner Chain due to it being subject to consensus.
```

Comment:
```suggestion
/// by all nodes participating in a partner chain due to it being subject to consensus.
```
I see we capitalize partner chain often but why? Isn't it a common noun?

Reply:
that's the preferred way of writing - I was correct on that before as well :)